src/test/components/ButtonWithPanel.test.js

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // @flow
-import React from 'react';
+import * as React from 'react';
```

Comment:
All these changes in tests are likely not needed as we don't use the types. But this was done automatically by the upgrade path so I thought I could keep them just in case. Tell me your thoughts.

Reply:
Might as well keep the auto-upgrade stuff.